internal/grid/handlers.go

```diff
@@ -62,7 +62,6 @@ const (
 	HandlerServerVerify
 	HandlerTrace
 	HandlerListen
-	HandlerGetLocalDiskIDs
```

Comment:
@harshavardhana You *cannot* remove handlers - this will create absolute havoc if it is connecting to an older version for example during a rolling upgrade.

There are 2 options:

1) Remove the server side completely and just rename it to `DeprecatedHandlerGetLocalDiskIDs` to retain the numbering. Any calls will return an error.

2) Keep it and provide a fallback on the server side that is "usable" for the remote. 

Either way you cannot safely remove it. Since you've released the damage is done.

Reply:
Yeah we don't recommend rolling upgrades anyway.

So customers have upgraded with no issues but I guess in future will be more careful here.